.github/workflows/pull-request.yml

```diff
@@ -304,7 +304,6 @@ jobs:
       - name: Upload Coverage to Codecov
         uses: codecov/codecov-action@v5
         with:
-          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This needs to be restored immediately. You have prevented our workflows from calculating the testing code coverage of the repository. It's an essential part of the stability of the code base. This is not acceptable.